### DISCUSSION TITLE
Previous page

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#6456] [Help]
The action navigate to ( window.history.back() ) does not work on the mobile phone browser.

**Comment by melohagan** [comment]
JavaScript in Budibase cannot access the **window** object.

**Comment by rafikApp** [comment]
What should i do ?

**Reply by melohagan** [reply]
There's an open issue here: https://github.com/Budibase/budibase/issues/5420

**Reply by rafikApp** [reply]
I want to pass the url of the current page( /somescreen/:id/:urlback ), is there a way to get the url or I have to add a column and compose it manually

**Reply by melohagan** [reply]
Hey @rafikApp currently you will need to manually build the URL.
